apps/eth_geth_adapter/config/prod.exs

```diff
@@ -0,0 +1,3 @@
+use Mix.Config
+
+config :ethereumex, url: "http://localhost:8545", client_type: :http
```

Comment:
Maybe we should ask Mana-Ethereum people to support `{:system, ...}` style config 😉